package.json

```diff
@@ -7,14 +7,17 @@
     "  3. Go to http://localhost:8000",
     "================================================================================"
   ],
-  "name": "@alaskaairux/auro-datetime",
+  "name": "@aurodesignsystem/auro-datetime",
```

Comment:
This is a BREAKING CHANGE

Reply:
Please update the comment for this commit in the body to include `BREAKING CHANGE`. Also, due to this namespace change, this CANNOT be shipped until the repo secrets are updated too. Once this work is approved I will update the secrets.